src/Illuminate/Foundation/Exceptions/Handler.php

```diff
@@ -212,22 +213,28 @@ protected function prepareResponse($request, Exception $e)
      *
      * @param  \Illuminate\Http\Request  $request
      * @param  \Exception $e
-     * @return \Symfony\Component\HttpFoundation\Response
+     * @return \Illuminate\Http\JsonResponse
      */
     protected function prepareJsonResponse($request, Exception $e)
     {
         $status = $this->isHttpException($e) ? $e->getStatusCode() : 500;
```

Comment:
Up to Taylor I guess? Those lines were what he wrote already in the master branch that currently exists. I'll leave it as is for this PR to keep changes to a minimum and we can review after?